src/main/MainScreen.js

```diff
@@ -22,12 +22,28 @@ export default class MainScreen extends PureComponent<Props> {
     this.props.navigation.navigate('DrawerOpen');
   };
 
+  constructor(props, context) {
+    super(props, context);
+    this.state = { shouldShowStatusBar: (
+      Dimensions.get('window').width < Dimensions.get('window').height
```

Comment:
you can get current orientation from app state.
like https://github.com/zulip/zulip-mobile/blob/master/src/account-info/AccountDetailsScreen.js#L48